e2e_tests/tests/experiment/test_pytorch.py

```diff
@@ -97,6 +97,9 @@ def test_pytorch_const_native_parallel() -> None:
 @pytest.mark.parametrize("aggregation_frequency", [1, 4])  # type: ignore
 @pytest.mark.parametrize("use_amp", [True, False])  # type: ignore
 def test_pytorch_const_parallel(aggregation_frequency: int, use_amp: bool) -> None:
```

Comment:
non-blocking: maybe worth considering setting the test suite up like below

https://github.com/brain-good/determined/blob/8b1d50b5b3484c36a3bdad80046c7eb060b4e0bd/e2e_tests/tests/experiment/test_tf_estimator.py#L85-L91